### DISCUSSION TITLE
Linter giving warning in `get` or `where` method

### DISCUSSION THREAD

**Discussion by sumitsharansatsangi** [piccolo-orm/piccolo#760] [General]
Tried Code:
```python

manager = await Manager.objects().get(Manager.name=="Sumit")

band = await Band.objects().get(Band.manager == manager.id)
```
Linter is giving following error

` Expected type 'Where | WhereRaw | And | Or', got 'bool | int' instead `

If , I'll provide a hard coded value like as follows, Linter does not gives any error.
```python

manager = await Manager.objects().get(Manager.name=="Sumit")

band = await Band.objects().get(Band.manager == 4)
```
I tried following too , but doesn't work.
```python

manager: Manager = await Manager.objects().get(Manager.name=="Sumit")

band: Band = await Band.objects().get(Band.manager == manager.id)
```

`I am using Pycharm as editor, mypy and black as linter`

**Comment by dantownsend** [comment]
Remember to use `==`. So this:

```python
manager: Manager = await Manager.objects().get(Manager.name="Sumit")
```

Should be:

```python
manager: Manager = await Manager.objects().get(Manager.name == "Sumit")
```

**Reply by sumitsharansatsangi** [reply]
Ah, it was `manager: Manager = await Manager.objects().get(Manager.name == "Sumit")` .
`=` was a type mistake happened writing this discussion,.
The warning is coming with 
```python

manager: Manager = await Manager.objects().get(Manager.name=="Sumit")

band: Band = await Band.objects().get(Band.manager == manager.id)
```
and 
```python

manager = await Manager.objects().get(Manager.name=="Sumit")

band = await Band.objects().get(Band.manager == manager.id)
```

The warning highlight `Band.manager == manager.id` part 

Code is working fine, but editor looks ugly, when linter highlights the warning part.

**Reply by stronk7** [reply]
May be related / the same than #436, for the records.

I'm also using PyCharm here and ALL the `where()` conditions are highlighting the content with `Expected type 'Where | WhereRaw | And | Or | QueryString', got 'bool' instead` warning (no matter the code is working perfectly).
